core/src/commonMain/kotlin/entity/interaction/Interaction.kt

```diff
@@ -24,7 +24,7 @@ public sealed interface Interaction : InteractionBehavior {
 
     override val applicationId: Snowflake get() = data.applicationId
 
-    override val channelId: Snowflake get() = data.channelId
+    public override val channelId: Snowflake? get() = data.channel.value?.id
```

Comment:
```suggestion
    override val channelId: Snowflake? get() = data.channel.value?.id ?: data.channelId.value
```
use the old `channel_id` field as a fallback